growth/too/models.py

```diff
@@ -830,11 +830,9 @@ class Observation(db.Model):
         primary_key=True,
         comment='Observation ID')
 
-    dateobs = db.Column(
+    expobs = db.Column(
```

Comment:
Why the name change?

Reply:
dateobs is the event name..

Reply:
... Shit! OK, could we get that in a separate pull request, please?